cmd/update.go

```diff
@@ -81,30 +85,37 @@ func updateDNS(c Config) (bool, string) {
 
 	encodedPayload, err := json.Marshal(payload)
 	if err != nil {
-		return false, err.Error()
+		return err
 	}
+
 	req := bytes.NewBuffer(encodedPayload)
 
 	res, err := http.Post(endpoint, "application/json", req)
 	if err != nil {
-		return false, err.Error()
+		return err
 	}
+
 	defer res.Body.Close()
 
 	body, err := io.ReadAll(res.Body)
 	if err != nil {
-		return false, err.Error()
+		return err
 	}
+
 	json.Unmarshal(body, &result)
 
-	if result.Status == "SUCCESS" || result.Message == "Edit error: We were unable to edit the DNS record." {
-		return true, result.Message
-	} else {
-		return false, result.Message
+	if result.Status == "ERROR" {
+		return fmt.Errorf("error updating dns")
```

Comment:
Since it's hard to identify what as the issue, maybe a message flagging what may be wrong would be better. The error that might happen on this line is probably because the config was not set up correctly.